components/apimgt/org.wso2.carbon.apimgt.rest.api.publisher.v1/src/main/java/org/wso2/carbon/apimgt/rest/api/publisher/v1/impl/ApisApiServiceImpl.java

```diff
@@ -2160,14 +2244,13 @@ public Response addAPIMediationPolicy(String apiId, String type, String ifMatch,
                 OMElement seqElement = APIUtil.buildOMElement(new ByteArrayInputStream(sequenceBytes));
                 String localName = seqElement.getLocalName();
                 fileName = seqElement.getAttributeValue(new QName("name"));
-                
                 if (APIConstants.MEDIATION_SEQUENCE_ELEM.equals(localName)) {
                     Mediation mediationPolicy = new Mediation();
                     mediationPolicy.setConfig(content);
-                    mediationPolicy.setName(fileName);
+                    mediationPolicy.setName(localName);
```

Comment:
This change done due to a different issue?

Reply:
It need not be changed. Reverted back to the original version. https://github.com/wso2/carbon-apimgt/pull/9609/commits/20a6fb69a612c320d26f57d1d19519722fe30ab5